engines/director/resource.cpp

```diff
@@ -202,8 +202,16 @@ Archive *Window::openMainArchive(const Common::String movie) {
 		delete _mainArchive;
 		_mainArchive = nullptr;
 
-		warning("openMainArchive(): Could not open '%s'", movie.c_str());
-		return nullptr;
+		warning("openMainArchive(): Could not open '%s'; trying EXE", movie.c_str());
+
+		if (g_director->getPlatform() == Common::kPlatformWindows) {
```

Comment:
Please check the file for existence before this.

Also, `loadEXE()` has to be changed too because it calls `error()` in case of failure. Let it return `false` or just rely on `_macArchive == nullptr` as it happens now.

Reply:
Added a file existence check. Confirmed it's working on both the Mac and Windows ports.